toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1433,6 +1426,15 @@ func provisionUserSSHCerts(installChroot *safechroot.Chroot, user configuration.
 	return
 }
 
+func cleanupExtraFiles(fileName string) (err error) {
+	err = os.Remove(fileName)
+	if err != nil {
+		logger.Log.Warnf("Failed to cleanup file (%s). Error: %s", fileName, err)
+		return
```

Comment:
Nit: no longer necessary, since we return immediately afterwards anyway.;)

Reply:
Removed, thanks